addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -509,7 +509,42 @@
                 </div>
             </we-input>
         </we-row>
-        <we-row class="o_strip_domain d-none" t-attf-class="#{widget.isButton ? ' d-none' : ''}">
+        <we-row>
+            <we-title>⌙ Media</we-title>
+            <div class="o_we_edit_file_wrapper">
+                <span class="text-truncate o_we_filename"><t t-out="widget.fileName"/></span>
+                <we-button class="o_we_bg_success o_we_upload_file" title="Replace file">
+                    <i class="fa fa-fw fa-refresh"/>
+                </we-button>
+                <we-button class="o_we_bg_danger o_we_remove_file" title="Remove file">
+                    <i class="fa fa-fw fa-trash"/>
+                </we-button>
+            </div>
+            <div class="o_we_upload_file_wrapper">
+                <we-button class="o_we_bg_success o_we_upload_file" title="Upload file">Upload a file</we-button>
+            </div>
+        </we-row>
+        <we-row class="o_we_file_action_wrapper">
+            <we-select class="o_we_user_value_widget">
+                <we-title>⌙ Action</we-title>
+                <div class="dropdown">
+                    <button class="dropdown-toggle"
+                        data-toggle="dropdown" title="" tabindex="-1"
+                        data-original-title="File Action" aria-expanded="false">
+                        <we-toggler>Open</we-toggler>
+                    </button>
+                    <we-selection-items name="file_action" class="dropdown-menu">
+                        <we-button t-attf-class="dropdown-item" href="#" data-value="download">
+                            Download
+                        </we-button>
+                        <we-button t-attf-class="dropdown-item" href="#" data-value="">
+                            Open
+                        </we-button>
+                    </we-selection-items>
+                </div>
+            </we-select>
+        </we-row>
+        <we-row class="o_strip_domain d-none" t-attf-class="o_strip_domain #{widget.isButton ? ' d-none' : ''}">
```

Comment:
Why adding `o_strip_domain` on t-attf-class?
One or the other should go, but you can't have both.
In qweb python, the t-attf-class will only be taken into consideration
In qweb js I wasnt sure but I just tested and it seems like the class will be used (so the opposite)
So you diff on this line actually does nothing if I am not wrong